pennylane/templates/state_preparations/state_prep_mps.py

```diff
@@ -15,24 +15,73 @@
 Contains the MPSPrep template.
 """
 
+import numpy as np
+
 import pennylane as qml
 from pennylane.operation import Operation
 from pennylane.wires import Wires
 
 
+def _complete_unitary(columns):
+    """
+    Completes a unitary matrix given a list of orthonormal columns.
```

Comment:
It might need some mathematical equation to depict the finally concluded algorithm used within.